metrics/nist_mt/nist_mt.py

```diff
@@ -0,0 +1,137 @@
+from typing import Dict, Optional
+
+import datasets
+import nltk
+from datasets import Sequence, Value
+
+
+try:
+    nltk.data.find("perluniprops")
+except LookupError:
+    nltk.download("perluniprops", quiet=True)  # NISTTokenizer requirement
+
+from nltk.tokenize.nist import NISTTokenizer
+from nltk.translate.nist_score import corpus_nist, sentence_nist
+
+import evaluate
+
+
+_CITATION = """\
+@inproceedings{10.5555/1289189.1289273,
+    author = {Doddington, George},
+    title = {Automatic Evaluation of Machine Translation Quality Using N-Gram Co-Occurrence Statistics},
+    year = {2002},
+    publisher = {Morgan Kaufmann Publishers Inc.},
+    address = {San Francisco, CA, USA},
+    booktitle = {Proceedings of the Second International Conference on Human Language Technology Research},
+    pages = {138–145},
+    numpages = {8},
+    location = {San Diego, California},
+    series = {HLT '02}
+}
+"""
+
+_DESCRIPTION = """\
+DARPA commissioned NIST to develop an MT evaluation facility based on the BLEU
+score. The official script used by NIST to compute BLEU and NIST score is
+mteval-14.pl. The main differences are:
+
+ - BLEU uses geometric mean of the ngram overlaps, NIST uses arithmetic mean.
+ - NIST has a different brevity penalty
+ - NIST score from mteval-14.pl has a self-contained tokenizer (in the Hugging Face implementation we rely on NLTK's
+implementation of the NIST-specific tokenizer)
+"""
+
+
+_KWARGS_DESCRIPTION = """
+Computes NIST score of translated segments against one or more references.
+Args:
+    predictions: predictions to score. For sentence-level NIST, a string;
+     for corpus-level NIST, a list of setences (str)
+    references:  potentially multiple references for each prediction.  For sentence-level NIST, a
+     list of potential references (str); for corpus-level NIST, a list (corpus) of lists
+     of potential references (str)
+    n: highest n-gram order
+    tokenize_kwargs: arguments passed to the tokenizer (see: https://github.com/nltk/nltk/blob/90fa546ea600194f2799ee51eaf1b729c128711e/nltk/tokenize/nist.py#L139)
+Returns:
+    'nist_mt': nist_mt score
+Examples:
+    >>> nist_mt = evaluate.load("nist_mt")
+    >>> hypothesis = "It is a guide to action which ensures that the military always obeys the commands of the party"
+    >>> reference1 = "It is a guide to action that ensures that the military will forever heed Party commands"
+    >>> reference2 = "It is the guiding principle which guarantees the military forces always being under the command of the Party"
+    >>> reference3 = "It is the practical guide for the army always to heed the directions of the party"
+    >>> nist_mt.compute(predictions=[hypothesis], references=[[reference1, reference2, reference3]])
+    {'nist_mt': 3.3709935957649324}
+"""
+
+
+@evaluate.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class Nist_mt(evaluate.Metric):
```

Comment:
I am a bit uneasy with that class name :) Can we make it 
```suggestion
class NistMt(evaluate.Metric):
```
or something similar? Class names are not supposed to have underscores.

Reply:
Sure! I don't like it either, not sure how that slipped in. My bad. I think you mentioned before that we should not just call it NIST because there are other NIST metrics out there, so I guess NistMt is the best option?